src/main/java/lotto/domain/LottoNumbers.java

```diff
@@ -0,0 +1,26 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoNumbers {
+
+    private static final List<LottoNumber> LOTTO_NUMBERS = initialLottoRange();
+
+    private static List<LottoNumber> initialLottoRange() {
+        List<LottoNumber> lottoNumbers = IntStream.range(1, 46).boxed().map(LottoNumber::new).collect(Collectors.toList());
```

Comment:
```suggestion
        List<LottoNumber> lottoNumbers = IntStream.range(1, 46).boxed().map(LottoNumber::new).collect(Collectors.toList());
```

1, 46 매직넘버네요 ~ 
지금은 훈련의 과정이니까 과하다싶을정도로 한번 해보는것도 좋을것 같아요! 😄

Reply:
매직 넘버 추가하면서 issueNumbers메서드에 subList 하는 부분에 LOTTO_SIZE를 추가해줬는데 LOTTO_SIZE는 Lotto 클래스로 들어가는게 맞다는 생각이 듭니다. 그런데 상수를 특별한 이유가 없다면 private으로 선언하고 클래스 내부에서만 사용하라고 하라는 말도 있던데 Lotto에 상수로 선언하고 다른곳에서 사용하는건 지양해야하나요?